tests/dummy/app/templates/application.hbs

```diff
@@ -39,4 +39,26 @@
     showCurrentIndicator=false
 }}
 
+{{input type='checkbox' checked=hideXAxisLines}}Hide x tick lines<br/>
+{{input type='checkbox' checked=showYTicks}}Show y ticks<br/>
+{{input type='checkbox' checked=showYGridLines}}Show y tick lines<br/>
+<button{{action "increaseQueue"}}>Increase</button>
+<button{{action "decreaseQueue"}}>Decrease</button>
+<div style='width: 20%'>
```

Comment:
nitpick but maybe we could make the example chart a little bigger? I thought something was wrong when I first opened it up because it was so small : )

Reply:
maybe like 90% or something?